ts/interactions/messageInteractions.ts

```diff
@@ -24,9 +28,14 @@ export function banUser(userToBan: string, conversationId: string) {
     window.inboxStore?.dispatch(updateConfirmModal(null));
   };
 
+  const title = deleteAllMessages ? window.i18n('banUserAndDeleteAll') : window.i18n('banUser');
+  const message = deleteAllMessages
+    ? window.i18n('banUserAndDeleteAllConfirm')
+    : window.i18n('banUserConfirm');
+
   const confirmationModalProps = {
-    title: window.i18n('banUser'),
-    message: window.i18n('banUserConfirm'),
+    title: title,
+    message: message,
```

Comment:
no need for the secont title and message here, if the field and the value have the same name you can just have

{
title,
message,
...
}

Reply:
Ok!
I fixed that :)